src/EventStore.Core/LogV2/LogV2RecordFactory.cs

```diff
@@ -47,7 +57,7 @@ public class LogV2RecordFactory : IRecordFactory<string> {
 				eventId: eventId,
 				transactionPosition: transactionPosition,
 				transactionOffset: transactionOffset,
-				eventStreamId: eventStreamId,
+				eventStreamId: eventStreamId ?? string.Empty,
```

Comment:
@pvanbuijtene this change looks strange, was the intended change `eventType: eventType ?? string.Empty` ?

Reply:
i'll remove this, perhaps it was for eventType but that will get converted to "" in the preparerecord itself anyway